frontend/src/scenes/billing/Billing.js

```diff
@@ -0,0 +1,174 @@
+                    </div>
+                )}
+            </Card>
+            <div className="space-top"></div>
+            <Card title="Billing plan">
+                {user.billing.plan && !user.billing.should_setup_billing && (
+                    <>
+                        Your organization is currently on the <b>{user.billing.plan.name}</b>. We're working on allowing
+                        self-serve billing management, in the meantime, please{' '}
+                        <a href="mailto:hey@posthog.com?subject=Billing%20management">contact us</a> if you wish to
+                        change or cancel your subscription.
+                    </>
+                )}
+                {user.billing.plan && user.billing.should_setup_billing && (
+                    <>
+                        Your organization is currently enrolled in the <b>{user.billing.plan.name}</b>, but billing
+                        details have not been set up. Please <a href={user.billing.subscription_url}>set them up now</a>{' '}
+                        or change your plan.{' '}
+                    </>
+                )}
+                {!user.billing.plan && <>Your organization does not have a billing plan set up yet.</>}
+                {plans?.results?.length > 0 && (
+                    <>
+                        Choose a plan from the list below to initiate a subscription.{' '}
+                        <b>
+                            For more information on our plans, check out our{' '}
+                            <a href="https://posthog.com/pricing" target="_blank">
+                                pricing page
+                            </a>
+                            .
+                        </b>
+                        <Row gutter={16} className="space-top">
+                            {plans.results.map((plan) => (
+                                <Col sm={24 / plans.results.length} key={plan.key} className="text-center">
+                                    {billingSubscriptionLoading && (
+                                        <Spin>
+                                            <Plan plan={plan} onUpgrade={handleBillingSubscribe} />
+                                        </Spin>
+                                    )}
+                                    {!billingSubscriptionLoading && (
+                                        <Plan plan={plan} onUpgrade={handleBillingSubscribe} />
+                                    )}
+                                </Col>
+                            ))}
+                        </Row>
+                    </>
+                )}
+            </Card>
+            <div style={{ marginBottom: 128 }}></div>
+        </>
+    )
+}
+
+Billing.propTypes = {
+    user: PropTypes.object.isRequired,
+}
+
+Plan.propTypes = {
+    plan: PropTypes.object.isRequired,
+    onUpgrade: PropTypes.func.isRequired,
+}
+
+export default Billing
```

Comment:
Same comment here regarding the default export and the proptypes as above with BillingToolbar